pkg/systemd/logs.js

```diff
@@ -288,6 +380,391 @@ $(function() {
+
+        var r_btn = $();
+        if (problem.IsReported){
+            for (var pid = 0; pid < problem.Reports.length; pid++) {
+                if (problem.Reports[pid][0] === 'ABRT Server') {
+                    var url = problem.Reports[pid][1]['URL']['v']['v'];
+                    r_btn = $('<a class="problem-btn">')
+                            .attr('href', url)
+                            .attr("target", "_blank")
+                            .text('Reported');
+                    break;
+                }
+            }
+        }
+
+        else if (problem.CanBeReported){
+            r_btn = $('<button class="btn btn-primary problem-btn">').text('Report');
+
+            r_btn.click(function() {
+                tab.children(':last-child').replaceWith($('<div class="spinner problem-btn">'));
+                var proc = cockpit.spawn(['reporter-ureport', '-d', problem.ID], { superuser: 'true' });
+                proc.done(function() {
+                    window.location.reload();
+                });
+                proc.fail(function(ex) {
+                    var message;
+                    if (ex.problem === 'access-denied') {
+                        message = _('Not authorized to upload-report');
+                    } else if (ex.problem === "not-found") {
+                        message = _('Reporter "reporter-ureport" not found.');
+                    } else {
+                        message = _('Reporting was unsucessful. Try running `reporter-ureport -d '+ problem.ID +'`');
+                    }
+                    $('<div class="alert alert-danger">')
+                        .append('<span class="pficon pficon-error-circle-o">')
+                        .text(message).insertAfter(".breadcrumb");
+                    tab.children(':last-child').replaceWith($('<span>'));
+                });
+            });
+        }
+
+        ge_t.click(function() {
+            switch_tab(ge_t, ge);
+        });
+
+        pi_t.click(function() {
+            switch_tab(pi_t, pi);
+        });
+
+        pd_t.click(function() {
+            switch_tab(pd_t, pd);
+        });
+
+        d_btn.click(function() {
+            service.DeleteProblems([problem.path]);
+            cockpit.location.go('/');
+        });
+
+        // write into general tab non-ABRT related items
+        var keys = Object.keys(entry).sort();
+        $.each(keys, function(i, key) {
+            if (key !== 'MESSAGE' && !key.startsWith('PROBLEM_')) {
+                ge.append(
+                    $('<tr>').append(
+                        $('<td>').css('text-align', 'right').text(key),
+                        $('<td>').css('text-align', 'left').
+                            text(journal.printable(entry[key]))));
+            }
+        });
+
+        tab.html(ge_t);
+        tab.append(pi_t);
+        tab.append(pd_t);
+        tab.append(d_btn);
+        tab.append(r_btn);
+
+        out.html(tab);
```

Comment:
`out` is a `<table>` and you can't put things like a `<ul>` in there.  Maybe put it within a `<caption>`?  It's probably best to put a number of  `<table>`s into the `<li>`s, though.